internal/server/stat/golden/get_stat_set_within_place/epa_facility.json

```diff
@@ -9,6 +9,7 @@
         },
         "epaGhgrpFacilityId/1000033": {
           "date": "2019",
+          "value": 0,
```

Comment:
Its believable that emissions (Annual_Emissions_GreenhouseGas_NonBiogenic) could be 0?

Has a downward trend: https://screenshot.googleplex.com/6Jq55JRgMKgknXq

Reply:
I checked this and other diffs, all looking good!